lib/addons/p5.dom.js

```diff
@@ -528,10 +528,11 @@
 
   /**
    * Creates a dropdown menu &lt;select&gt;&lt;/select&gt; element in the DOM.
-   * It also helps to asssign select-box methods to p5.Element when selecting existing select box
+   * It also helps to assign select-box methods to p5.Element when selecting existing select box
    * @method createSelect
-   * @param {boolean} [multiple] true if dropdown should support multiple selections
-   * @return {Object|p5.Element} pointer to p5.Element holding created node
+   * @param {boolean} [multiple] true if dropdown should support multiple selections OR
+   * @param {Object  [P5.Element] if wrapping methods to existing select
```

Comment:
Typo here, probably should be `{Object|p5.Element} if wrapping methods to existing select`